tutorials/hyperspectral/Advanced_Spectral_Signatures.Rmd

```diff
@@ -0,0 +1,291 @@
+---
+title: "Advanced Spectral Signatures in R"
+code1: HYPERSPECTRAL/Advanced_Spectral_Signatures.R
+contributors: null
+output:
+  html_document:
+    df_print: paged
+dateCreated: 2020-02-18 20:18:34
+description: Plot the spectral signatures of multiple different land cover types using
+  an interactive click-to-extract interface.
+estimatedTime: 0.5 Hours
+languagesTool: R
+dataProudct: null
```

Comment:
Is this not useful for working with our hyperspectral data?  If so, add the DP#.

Reply:
Good call, I added the data product ID for this dataset.